modelgrid_aliases_nuopc.xml

```diff
@@ -753,38 +610,196 @@
+    <grid name="lnd">0.9x1.25</grid>
+    <grid name="glc">gris4</grid>
+    <grid name="ocnice">gx1v7</grid>
+    <mask>gx1v7</mask>
+  </model_grid>
+  <model_grid alias="f10_f10_ais8_mg37" compset="_CISM|_DGLC" not_compset="_POP">
+    <grid name="atm">10x15</grid>
+    <grid name="lnd">10x15</grid>
+    <grid name="ocnice">10x15</grid>
+    <grid name="glc">ais8</grid>
+    <mask>gx3v7</mask>
+  </model_grid>
+  <model_grid alias="f10_f10_ais8gris4_mg37" compset="_CISM|_DGLC" not_compset="_POP">
+    <grid name="atm">10x15</grid>
+    <grid name="lnd">10x15</grid>
+    <grid name="ocnice">10x15</grid>
+    <grid name="glc">ais8:gris4</grid>
+    <mask>gx3v7</mask>
+  </model_grid>
+  <model_grid alias="f09_tn14_ais8" compset="_CISM|_DGLC" not_compset="_POP|_MOM">
+    <grid name="atm">0.9x1.25</grid>
+    <grid name="lnd">0.9x1.25</grid>
+    <grid name="glc">ais8</grid>
+    <grid name="ocnice">tnx1v4</grid>
+    <mask>tnx1v4</mask>
+  </model_grid>
+  <model_grid alias="f09_g17_ais8" compset="_CISM|_DGLC" not_compset="_BLOM">
+    <grid name="atm">0.9x1.25</grid>
+    <grid name="lnd">0.9x1.25</grid>
+    <grid name="glc">ais8</grid>
+    <grid name="ocnice">gx1v7</grid>
+    <mask>gx1v7</mask>
+  </model_grid>
+  <model_grid alias="f09_g17_ais8gris4" compset="_CISM|_DGLC" not_compset="_BLOM">
+    <grid name="atm">0.9x1.25</grid>
+    <grid name="lnd">0.9x1.25</grid>
+    <grid name="glc">ais8:gris4</grid>
+    <grid name="ocnice">gx1v7</grid>
+    <mask>gx1v7</mask>
+  </model_grid>
+  <model_grid alias="f09_tn14_ais8gris4" compset="_CISM|_DGLC" not_compset="_POP|_MOM">
+    <grid name="atm">0.9x1.25</grid>
+    <grid name="lnd">0.9x1.25</grid>
+    <grid name="glc">ais8:gris4</grid>
+    <grid name="ocnice">tnx1v4</grid>
+    <mask>tnx1v4</mask>
+  </model_grid>
+  <model_grid alias="f19_g17_ais8" compset="_CISM|_DGLC" not_compset="_BLOM">
+    <grid name="atm">1.9x2.5</grid>
+    <grid name="lnd">1.9x2.5</grid>
+    <grid name="ocnice">gx1v7</grid>
+    <grid name="glc">ais8</grid>
+    <mask>gx1v7</mask>
+  </model_grid>
+  <model_grid alias="f19_tn14_ais8" compset="_CISM|_DGLC" not_compset="_POP|_MOM">
+    <grid name="atm">1.9x2.5</grid>
+    <grid name="lnd">1.9x2.5</grid>
+    <grid name="glc">ais8</grid>
+    <grid name="ocnice">tnx1v4</grid>
+    <mask>tnx1v4</mask>
+  </model_grid>
+  <model_grid alias="f19_f19_ais8_mtn14" compset="_CISM|_DGLC" not_compset="_POP|_MOM">
+    <grid name="atm">1.9x2.5</grid>
+    <grid name="lnd">1.9x2.5</grid>
+    <grid name="glc">ais8</grid>
+    <grid name="ocnice">tnx1v4</grid>
+    <mask>tnx1v4</mask>
+  </model_grid>
+  <model_grid alias="f19_f19_ais8gris4_mtn14" compset="_CISM|_DGLC" not_compset="_POP|_MOM">
+    <grid name="atm">1.9x2.5</grid>
+    <grid name="lnd">1.9x2.5</grid>
+    <grid name="glc">ais8:gris4</grid>
+    <grid name="ocnice">tnx1v4</grid>
+    <mask>tnx1v4</mask>
+  </model_grid>
+  <model_grid alias="f19_tn14_gris4" compset="_CISM|_DGLC" not_compset="_POP|_MOM">>
```

Comment:
Invalid character that will cause xmllint to fail. Please run at least one test that validates the file (or just run it manually: `xmllint --schema ../cime/CIME/data/config/xml_schemas/config_grids_v2.2.xsd --noout modelgrid_aliases_nuopc.xml`).

Reply:
done.